windows/src/developer/TIKE/main/Keyman.Developer.UI.UframeCEFHost.pas

```diff
@@ -292,6 +425,7 @@ procedure TframeCEFHost.cefConsoleMessage(Sender: TObject;
   id: string;
   I: Integer;
 begin
+  AssertVclThread;
```

Comment:
Should this be `AssertCefThread`?

I think you used it in the other TframeCEFHost.cef* procedures

cefConsoleMessage

Reply:
I'm still confused if this should be `AssertVCLThread` or `AssertCefThread`

Reply:
Oops